awscli/examples/ssm/describe-instance-patches.rst

```diff
@@ -55,3 +55,30 @@ Output::
     ],
     "NextToken": "--token string truncated--"
   }
+
+**To get a list of patches installed patches since a specified InstalledTime for an instance**
```

Comment:
Looks like an extra word was accidentally added here.
```suggestion
**To get a list of patches installed since a specified InstalledTime for an instance**
```

Reply:
Removed extra word in latest rebase